scope.go

```diff
@@ -21,12 +21,18 @@
 package tally
 
 import (
+	"errors"
 	"fmt"
 	"sync"
+	"time"
 
 	"github.com/facebookgo/clock"
 )
 
+var (
+	errCloneNonRoot = errors.New("Called Close on non-root Scope")
```

Comment:
We could create a new interface called RootScope which inherits Scope to avoid this perhaps?